spec/support/common_actions.rb

```diff
@@ -357,4 +357,10 @@ def nested_path_for(resource)
     end
   end
 
+  def wait_for_ajax
```

Comment:
Just before we forget about it.. (not saying this should be fixed now) But as we talked while dealing with flakies... maybe there's an alternative to the usage of `wait_for_ajax` as you did at https://github.com/AyuntamientoMadrid/consul/pull/1206 🤔

Reply:
This flaky is related to this https://github.com/AyuntamientoMadrid/consul/issues/1194, and I'm looking for a solution in Madrid's fork.